coderedcms/templates/coderedcms/blocks/card_foot.html

```diff
@@ -9,9 +9,9 @@
     {% if self.subtitle %}<h6 class="card-subtitle mb-2 text-muted">{{self.subtitle}}</h6>{% endif %}
     <div class="card-text">{{self.description}}</div>
   </div>
-  <div class="card-footer">
-    {% for button in self.links %}
-    {% include_block button %}
-    {% endfor %}
-  </div>
+  {% for button in self.links %}
```

Comment:
I think it would be best to leave the for-loop inside the div, and replace this with an `{% if self.links %}` check. 

For example, if there are two buttons, they should render side by side. This change would make them render in two separate footers.